docs: add "Graceful Shutdown" section to the quickstart (#11244)
VictoriaMetrics/VictoriaMetrics: docs/victoriametrics/Quick-Start.md

For backup configuration, please refer to [vmbackup documentation](https://docs.victoriametrics.com/victoriametrics/vmbackup/).

### Graceful shutdown

> To gracefully shut down VictoriaMetrics (or any other) process - send `SIGTERM` or `SIGINT` signals and wait until shutdown finishes.

Reviewer: ```suggestion
To gracefully shut down a VictoriaMetrics process, send SIGTERM or SIGINT and wait until the process exits.
```
Let's remove this and any other strange processes.

See [how to send signals to processes](https://stackoverflow.com/questions/33239959/send-signal-to-process-from-command-line).

> Graceful shutdown guarantees data safety and graceful handling of the ongoing connections.

Author: This change removes very important `guarantees data safety`

> Graceful shutdown guarantees data safety and graceful handling of the ongoing connections.

Reviewer: ```suggestion
Graceful shutdown is required for data safety. A successful graceful shutdown guarantees that pending in-memory data is flushed before the process exits. It stops accepting new HTTP connections, waits for in-flight requests, and cancels the remaining requests shortly before `-http.maxGracefulShutdownDuration` expires.
```
From my point of view, it is important to describe how it will affect requests and data. Not a blocker
